README.md

```diff
@@ -18,6 +18,10 @@ application to capture and report distributed traces to Splunk APM.
 > It is **officially supported** by Splunk.
 > However, breaking changes **MAY** be introduced.
 
+If you're currently using the SignalFx Go Agent and want to migrate to the
```

Comment:
```suggestion
If you're currently using the [SignalFx Tracing Library for Go](https://github.com/signalfx/signalfx-go-tracing)
and want to migrate to the
```

like in https://github.com/signalfx/splunk-otel-python

I never saw "SignalFx Go Agent" anywhere 😉